### DISCUSSION TITLE
Open GL ES2

### DISCUSSION THREAD

**Discussion by akulamartin** [GameUpCell2/IntoTheVoid#27] [Ideas]
@TobeTek Add support for es2 devices in the Android build. This version is still a big market in google playstore.

**Comment by TobeTek** [comment]
Thanks @akulamartin. I'll update the Player Settings appropriately to support more Graphic APIs. 
[Reference thread](https://answers.unity.com/questions/1041522/where-is-the-android-graphics-level-setting.html)
